setup.cfg

```diff
@@ -1,2 +1,2 @@
 [flake8]
-max-line-length = 88
+max-line-length = 120
```

Comment:
Is there a good reason to override this here? I think we use the default 88 in our other projects.